app/Transformers/UserTransformer.php

```diff
@@ -96,6 +96,7 @@ public function transform(User $user)
             'post_count' => $user->user_posts,
             'profile_colour' => $user->user_colour,
             'profile_order' => $profileCustomization->extras_order,
+            'ranking_expanded' => $profileCustomization->ranking_expanded,
```

Comment:
should be using the preferences of the user viewing the page, not the user of the page viewed